js/piwik.js

```diff
@@ -7152,16 +7187,16 @@ if (typeof window.Piwik !== 'object') {
         }
     }
 
-    window.Piwik.trigger('PiwikInitialized', []);
-    window.Piwik.initialized = true;
+    window.Matomo.trigger('MatomoInitialized', []);
```

Comment:
`PiwikInitialized` was never really documented or supported until few weeks ago so we can easily break BC